scripts/translation/lib/RevcheckIgnore.php

```diff
@@ -57,13 +57,6 @@ public static function ignore( $filename ) : bool
         if ( $filename == "reference/datetime/timezones.xml" )
             return true;
 
-        if ( IGNORE_EXTENSIONS_XML )
-            if ( str_ends_with( $filename , '/extensions.xml') ) // track/count backport
-                return true;
```

Comment:
Is it possible to hardcode this like the `reference/datetime/timezones.xml` file as:
```php
if ( $filename == "appendices/extensions.xml" )
            return true;
```
?

Reply:
I'm trying to make exceptions as few as possible. And in this case, I also think that the original ignore was as mistake:
```
$ find * | grep "/extensions.xml"
appendices/migration56/extensions.xml
appendices/extensions.xml
```
All files ending in "/extensions.xml" are translatable.

Reply:
`appendices/extensions.xml` should NOT be translated, it is a generated file. Which is identical to `reference/datetime/timezones.xml` this is why I asked to hardcode the exception.

I agree that not marking `appendices/migration56/extensions.xml` as translatable is a bug.

Reply:
Oh, I see the confusion. `appendices/extensions.xml` is already ignored (see line 53), and removing the blank ignore on `/extensions.xml` will not change this. So it's already done.